cpp/modules/luma.gl/garrow/src/array.cc

```diff
@@ -34,8 +34,15 @@ Array::~Array() {
 }
 
 void Array::setData(const std::shared_ptr<arrow::Array>& data) {
+  auto vertexFormatOptional = vertexFormatFromArrowType(data->type());
```

Comment:
- Interesting, here we accept a standard arrow array.
- I assume then we could also accept a chunked array?

Reply:
I see this as an intermediate solution that's here for convenience. As noted in TODO at the top of the file, in `arrow`, these are constructed using the builder API. As there's a lot of complexity in providing typed array types, and that kind of an API, this is a quick solution to filling the buffers using the data we already have